### PR TITLE
Chart Component Improvement

### PR DESCRIPTION
**Pull request by rodiguif**
## Description
This PR implements improvements on the chart component, creating separated attributes for each element.

### CODE REVIEW

File: app/src/main/java/com/oxeanbits/forecastchart/ui/MainExample.kt [hunk 1]
```diff
@@ -30,23 +35,25 @@ class MainExample : AppCompatActivity() {
...
-                            SetupChartExample.getForecastedObj(),
-                            SetupChartExample.getEndDateObj(),
-                            "m³"
-                        )
+                        actualLine(actualLine.values, actualLine.label,
+                            actualLine.color, actualLine.forecasted)
+                        forecastedLine(forecastedLine.values, forecastedLine.label,
+                            forecastedLine.color, forecastedLine.forecasted)
+                        endDateBar(endDateBar.x, endDateBar.y)
+                        unit("m³")
```
Comment (tiagogoncalves) [inline app/src/main/java/com/oxeanbits/forecastchart/ui/MainExample.kt:56]: Why this unit is hard coded?

**Review by tiagogoncalves — CHANGES_REQUESTED** [review]
just one question...

File: app/src/main/java/com/oxeanbits/forecastchart/ui/MainExample.kt (hunk 1 above)
Comment (rodiguif) [reply app/src/main/java/com/oxeanbits/forecastchart/ui/MainExample.kt:56]: That's is the Example.

**Review by tiagogoncalves — APPROVED** [review]
:100:

**Pull request merged**